scene/debugger/scene_debugger.cpp

```diff
@@ -1881,6 +1885,57 @@ void RuntimeNodeSelect::_physics_frame() {
 #endif // _3D_DISABLED
 	}
 
+	if ((prefer_group_selection || avoid_locked_nodes) && !list_shortcut_pressed && node_select_mode == SELECT_MODE_SINGLE) {
+		for (int i = 0; i < items.size(); i++) {
+			Node *node = items[i].item;
+			Node *final_node = node;
+			real_t order = items[i].order;
+
+			// Replace the node by the group if grouped.
+			if (prefer_group_selection) {
+				while (node && node != root) {
+					if (node->has_meta("_edit_group_")) {
+						final_node = node;
+
+						if (Object::cast_to<CanvasItem>(final_node)) {
+							CanvasItem *ci_tmp = Object::cast_to<CanvasItem>(final_node);
+							order = ci_tmp->get_effective_z_index() + ci_tmp->get_canvas_layer();
+						} else if (Object::cast_to<Node3D>(final_node)) {
```

Comment:
This broke 3D disabled builds, needs to be in `#ifndef _3D_DISABLED` checks, will make a fix tomorrow if someone hasn't gotten to it before then

Edit: Will write a fix now, pushing momentarily